### ISSUE TITLE
megatron lora训练qwen3-omni速度很慢

### ISSUE THREAD

**Issue by yangyyt** [modelscope/ms-swift#7832]
### Checklist / 检查清单

- [x] I have searched existing issues, and this is a new bug report. / 我已经搜索过现有的 issues，确认这是一个新的 bug report。

### Bug Description / Bug 描述

使用最近的3.12的镜像训练qwen3omni模型，速度很慢回事哪些方面的影响？平均8min一个迭代：
[2026-01-20 08:33:58] iteration        5/     206 | consumed samples:          320 | elapsed time per iteration (ms): 100512.9 | memory(GiB): 21.54 | elapsed time: 10m 7s | remaining time: 6h 47m 4s | learning rate: 4.854369E-06 | global batch size:    64 | lm loss: 1.180399E+00 | load_balancing_loss: 1.268389E+00 | loss scale: 1.0 | grad norm: 1.105 | number of skipped iterations:   0 | number of nan iterations:   0 |
 [2026-01-20 08:42:14] iteration       10/     206 | consumed samples:          640 | elapsed time per iteration (ms): 99158.9 | memory(GiB): 21.56 | elapsed time: 18m 23s | remaining time: 6h 0m 26s | learning rate: 9.708738E-06 | global batch size:    64 | lm loss: 1.190541E+00 | load_balancing_loss: 1.278186E+00 | loss scale: 1.0 | grad norm: 0.826 | number of skipped iterations:   0 | number of nan iterations:   0 |
 [2026-01-20 08:50:29] iteration       15/     206 | consumed samples:          960 | elapsed time per iteration (ms): 98998.0 | memory(GiB): 21.56 | elapsed time: 26m 38s | remaining time: 5h 39m 12s | learning rate: 9.987198E-06 | global batch size:    64 | lm loss: 1.178630E+00 | load_balancing_loss: 1.273939E+00 | loss scale: 1.0 | grad norm: 0.978 | number of skipped iterations:   0 | number of nan iterations:   0 |
 [2026-01-20 08:58:56] iteration       20/     206 | consumed samples:         1280 | elapsed time per iteration (ms): 101551.1 | memory(GiB): 21.6 | elapsed time: 35m 6s | remaining time: 5h 26m 26s | learning rate: 9.945554E-06 | global batch size:    64 | lm loss: 1.061674E+00 | load_balancing_loss: 1.264268E+00 | loss scale: 1.0 | grad norm: 0.745 | number of skipped iterations:   0 | number of nan iterations:   0 |
 [2026-01-20 09:07:21] iteration       25/     206 | consumed samples:         1600 | elapsed time per iteration (ms): 100988.3 | memory(GiB): 21.6 | elapsed time: 43m 31s | remaining time: 5h 15m 4s | learning rate: 9.875285E-06 | global batch size:    64 | lm loss: 9.960416E-01 | load_balancing_loss: 1.263143E+00 | loss scale: 1.0 | grad norm: 0.519 | number of skipped iterations:   0 | number of nan iterations:   0 |

### How to Reproduce / 如何复现

megatron sft \
    --model ${model} \
    --load_safetensors true \
    --save_safetensors true \
    --merge_lora false \
    --dataset ${data_json} \
    --train_type lora \
    --lora_rank 8 \
    --lora_alpha 32 \
    --target_modules all-linear \
    --load_from_cache_file true \
    --split_dataset_ratio 0.01 \
    --moe_permute_fusion true \
    --pipeline_model_parallel_size 2 \
    --tensor_model_parallel_size 2 \
    --expert_model_parallel_size 2 \
    --decoder_first_pipeline_num_layers 25 \
    --moe_grouped_gemm true \
    --moe_shared_expert_overlap true \
    --moe_aux_loss_coeff 1e-6 \
    --micro_batch_size 1 \
    --global_batch_size 64 \
    --recompute_granularity full \
    --recompute_method uniform \
    --recompute_num_layers 1 \
    --max_epochs 1 \
    --finetune true \
    --cross_entropy_loss_fusion true \
    --lr 1e-5 \
    --lr_warmup_fraction 0.05 \
    --min_lr 1e-6 \
    --save megatron_output/${exp_name} \
    --eval_interval 500 \
    --save_interval 500 \
    --max_length 8192 \
    --packing true \
    --num_workers 8 \
    --dataset_num_proc 8 \
    --no_save_optim true \
    --no_save_rng true \
    --sequence_parallel true \
    --moe_expert_capacity_factor 2 \
    --attention_backend flash

### Additional Information / 补充信息

**Comment by Jintao-Huang** [comment]
https://swift.readthedocs.io/zh-cn/v3.12/BestPractices/Qwen3-VL-Best-Practice.html

<img width="693" height="205" alt="Image" src="https://github.com/user-attachments/assets/e88f8838-38a9-49e1-b746-2defa83386fa" />
